build.gradle

```diff
@@ -120,7 +120,6 @@ dependencies {
     constraints {
         implementation('org.scala-lang:scala-library:2.13.10')
         implementation('org.json:json:20230227')
-        implementation('org.bitbucket.b_c:jose4j:0.9.3')
```

Comment:
Can remove this and it remains [at 0.9.3](https://github.com/DataBiosphere/terra-common-lib/pull/113/files#diff-66edc8269e42fa41dfa043929a8580e710fa57e310fd6ac88ba04a4b20e23efeR157) after executing --write-locks.